src/lib/config/defaultConfig.js

```diff
@@ -25,6 +25,7 @@ export const APP_CONFIG = {
       className: '',
     },
   },
+  LOGOUT_METHOD: 'GET',
```

Comment:
It is not clear to me, when should I use `GET` and when `POST` for logout in my app.

Reply:
I kind of agree that this is tricky, in the sense that it will actually depend on how the user logged in, and not a global option. I have added a comment about this in `invenio-oauthclient`, we should figure it out there first.